### PR TITLE
Fixes #34951 - Pin @babel version to 7.17.10

### PR DESCRIPTION
**Pull request by MariaAga**
Needs https://github.com/theforeman/foreman-js/pull/406

### CODE REVIEW

**Comment by evgeni** [comment]
Do we *need* `7.17.10`, or do we need a version earlier than `7.17.12` (according to the redmine). I am asking because we do have 7.7.4 packaged right now, and that would either need updating, or not.

**Comment by MariaAga** [comment]
Packaging can keep `7.7.4`, we need something before `7.17.12`

File: package.json [hunk 1]
```diff
@@ -24,19 +24,19 @@
...
     "react-intl": "^2.8.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.7.0",
-    "@theforeman/builder": "^10.1.0",
-    "@theforeman/eslint-plugin-foreman": "^10.1.0",
-    "@theforeman/stories": "^10.1.0",
-    "@theforeman/test": "^10.1.0",
-    "@theforeman/vendor-dev": "^10.1.0",
+    "@babel/core": "7.17.10",
```
Comment (evgeni) [inline package.json:34]:
```suggestion
    "@babel/core": ">=7.7.0 <7.17.12",
```

this will do what you want, w/o freaking out packaging

**Comment by MariaAga** [comment]
Fixed the tests without this

**Pull request closed without merging**